package.json

```diff
@@ -17,13 +19,23 @@
   },
   "homepage": "https://github.com/anastasiya-nd/budget#readme",
   "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "@babel/preset-react": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^6.2.0",
     "editor": "^1.0.0",
```

Comment:
What for this package?

Reply:
I don't understand what you are saying.

Reply:
Why did you install this package (`editor`)?

Reply:
I thought I had to use eslint and editor together

Reply:
What is editor?